dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/DeepStorageBrowserPresenter.java

```diff
@@ -163,7 +164,16 @@ public void initialize(final URL location, final ResourceBundle resources) {
             initJobsPane();
             initDs3TreeTableView();
 
-            inlineCssTextArea.focusedProperty().addListener((observable, oldValue, newValue) -> selectAllItem.setDisable(oldValue));
+            logView.focusedProperty().addListener((observable, oldValue, newValue) -> selectAllItem.setDisable(oldValue));
+            logView.setFocusTraversable(false);
+            logView.getItems().addListener((ListChangeListener<? super Text>) c -> {
+                final ObservableList<? extends Text> cList = c.getList();
```

Comment:
This seems like this could be a race condition depending on which thread this handler runs in.  Do we know if this will run in the ui thread or not?